v2/cn/ann-1019009001rc1.md.tt2

```diff
@@ -0,0 +1,108 @@
+[% VER = "1.19.9.1"; RC = 1 -%]
+[% major_ver = '1.19.9' -%]
+<!---
+    @title         OpenResty [% VER %] RC[% RC %] is out
+--->
+
+We have just kicked out a new release candidate, OpenResty [% VER %] RC[% RC %],
+for the community to test out.
+
+# Source package
+
+https://openresty.org/download/openresty-[% VER %]rc[% RC %].tar.gz
+
+PGP for this source tarball:
+
+https://openresty.org/download/openresty-[% VER %]rc[% RC %].tar.gz.asc
+
+# Windows binary packages
+
+## Win64 version:
+
+https://openresty.org/download/openresty-[% VER %]rc[% RC %]-win64.zip
+
+PGP for the Win64 zip file:
+
+https://openresty.org/download/openresty-[% VER %]rc[% RC %]-win64.zip.asc
+
+## Win32 version:
+
+https://openresty.org/download/openresty-[% VER %]rc[% RC %]-win32.zip
+
+PGP for the Win32 zip file:
+
+https://openresty.org/download/openresty-[% VER %]rc[% RC %]-win32.zip.asc
+
+# Acknowledgments
+
+Special thanks go to all our developers and contributors! Also thanks Junlong Li,
+and Jiahao Wang for their help in preparing this release.
+
+# Version highlights
+
+The highlights of this release candidate are:
+
+* Based on the very recent mainline [NGINX](nginx.html) core [% major_ver %].
+* Many fixes imported from Mike Pall's upstream [LuaJIT](https://github.com/openresty/luajit2#readme) repository...
+* When lua failed in requesting memory, instead of graceful shutdown, `abort()` is called.
+* Now the `get_ctx_table` supports using the ctx table from caller, and that will reduce the cost of creating a new ctx table.
+* For a better performance when using [lua-tablepool](https://github.com/openresty/lua-tablepool), discard the objects when the pool size is larger than the max_pool_size.
+* Implemented process API for stream subsystem.
```

Comment:
What is the "process API". Are you referring to `ngx.process`? Better be more explicit here and also add a link to it.